charts/tailscale-subnet-router/templates/statefulset.yaml

```diff
@@ -29,7 +29,7 @@ spec:
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       containers:
-        - name: {{ .Chart.Name }}
+        - name: {{ include "tailscale-subnet-router.fullname" . }}
```

Comment:
I'd prefer to keep this as the shorter chart name, since the fullname isn't necessary in this context.

Reply:
@gtaylor Thanks for the review. Would you consider using `tailscale-subnet-router.name` instead (which defaults to `Chart.Name`)? I would like to be able to override it since we plan on having several exit nodes deployed in multiple environments.

Reply:
Yes, that sounds good!